bin/katello-certs-check

```diff
@@ -45,8 +48,11 @@ if [ -z "$CERT_FILE" -o -z "$KEY_FILE" -o -z "$CA_BUNDLE_FILE" ]; then
     exit 1
 fi
 
-HOSTNAME=$(hostname -f)
-CERT_HOSTNAME=$(openssl x509 -noout -subject -in $CERT_FILE | sed -e 's/^subject.*CN=\([a-zA-Z0-9\.\-]*\).*$/\1/')
+if [[ $TARGET != "foreman" && $TARGET != "foreman-proxy" ]]; then
+    echo 'Target not specified or incorrect' >&2
```

Comment:
The target usage doesn't tell the user that only `foreman` and `foreman-proxy` are the only valid values. The user now has no way to know.